src/test/java/org/fcrepo/transform/transformations/LDPathTransformTest.java

```diff
@@ -141,35 +143,39 @@ public void testGetNodeTypeSpecificLdpathProgramForNodeTypeProgram() throws Repo
         final FedoraBinary mockChildConfig = mock(FedoraBinary.class);
         when(mockChildConfig.getPath())
                 .thenReturn(CONFIGURATION_FOLDER + "some-program/" + customNsPrefix + ":type");
-        when(mockConfigNode.getChildren()).thenReturn(Stream.of(mockChildConfig));
+        when(mockConfigNode.getChildren()).thenReturn(of(mockChildConfig));
 
         final URI mockRdfType = UriBuilder.fromUri(customNsUri + "type").build();
         when(mockResource.getTypes()).thenReturn(Arrays.asList(mockRdfType));
 
         when(mockChildConfig.getContent()).thenReturn(mockInputStream);
 
-        final LDPathTransform nodeTypeSpecificLdpathProgramStream =
-                getResourceTransform(mockResource, mockSession, mockNodeService, "some-program");
+        final LDPathTransform nodeTypeSpecificLdpathProgramStream = getResourceTransform(mockResource, mockSession,
+                mockNodeService, mockBackend, "some-program");
 
-        assertEquals(new LDPathTransform(mockInputStream), nodeTypeSpecificLdpathProgramStream);
+        assertEquals(new LDPathTransform(mockBackend, mockInputStream),
+                nodeTypeSpecificLdpathProgramStream);
     }
 
     @Test
     public void testProgramQuery() {
 
-        final RdfStream rdfStream = new DefaultRdfStream(createURI("abc"), of(
-                create(createURI("abc"),
-                        createURI("http://purl.org/dc/elements/1.1/title"),
-                        createLiteral("some-title"))));
-        final InputStream testReader = new ByteArrayInputStream("title = dc:title :: xsd:string ;".getBytes());
+        final RdfStream rdfStream = new DefaultRdfStream(
+                createURI("http://purl.org/dc/elements/1.1/contributor"), empty());
+        final InputStream testReader = new ByteArrayInputStream("title = rdfs:label :: xsd:string ;".getBytes());
 
-        testObj = new LDPathTransform(testReader);
+        testObj = new LDPathTransform(mockBackend, testReader);
         final List<Map<String,Collection<Object>>> stringCollectionMapList = testObj.apply(rdfStream);
+
         final Map<String,Collection<Object>> stringCollectionMap = stringCollectionMapList.get(0);
 
         assert(stringCollectionMap != null);
+
         assertEquals(1, stringCollectionMap.size());
+
+        assumeTrue(stringCollectionMap.get("title").size() > 0);
```

Comment:
This involves a network call to an endpoint that isn't always available. I would highly recommend leaving this as is. Otherwise, there will be transient errors unrelated to our code